deeplake/integrations/pytorch/shuffle_buffer.py

```diff
@@ -92,9 +92,13 @@ def emtpy(self) -> bool:
         return len(self.buffer) == 0
 
     def _sample_size(self, sample):
-        if isinstance(sample, bytes):
+        if isinstance(sample, (int, float)):
```

Comment:
how are we getting this? are there cases in which dataloader will now return int/float/bool?